l3kernel/l3tl.dtx

```diff
@@ -2611,6 +2723,29 @@
 %    \end{macrocode}
 % \end{macro}
 %
+% \begin{macro}
+%   {
+%     \tl_if_regex_match:nn, \tl_if_regex_match:Vn,
+%     \tl_if_regex_match:nN, \tl_if_regex_match:VN,
+%   }
+%    \begin{macrocode}
+\prg_new_protected_conditional:Npnn \tl_if_regex_match:nn #1#2 { TF , T , F }
+  {
+    \regex_match:nnTF {#2} {#1}
+      \prg_return_true: \prg_return_false:
+  }
```

Comment:
Wondering why `\regex_match:nn(TF)` is not called `\regex_if_match:nn(TF)`.

Reply:
good point. would be more consistent